server/routes/caseNotes/viewAll/caseNotesPresenter.ts

```diff
@@ -61,6 +62,7 @@ export default class CaseNotesPresenter {
       sentAtDate: DateUtils.formattedDate(caseNote.sentAt),
       sentAtTime: DateUtils.formattedTime(caseNote.sentAt),
       sentBy: officerName,
+      sentByUserType: caseNote.sentBy.authSource === 'delius' ? 'probation practitioner' : 'service provider',
```

Comment:
Do you think it'd be a good idea to wrap this `authSource === 'delius'` in a helper function somewhere? We currently repeat this check in a few places:

- server/routes/index.ts
- mockApis/auth.ts
- server/services/hmppsAuthService.ts

I feel like encapsulating it in an `isProbationPractitioner` function would hide the internals about an `auth_source` field that comes back from the API, so we could forget about it, and we'd only need to update it in one place if this value changed.

Ideally, I feel this would live on a `User` model, but we don't tend to do much manipulation of the values we get from the API. We've tended to solve this sort of thing with `Decorator`-type classes in the past, but maybe for now a helper function would do.

I don't think it should block this PR, we could do it in a separate tech-debt style card.

Reply:
Yeah, I agree. I'll raise it as a new PR? 
We're running out of tickets to do for this sprint so will pick it up after this.